docs/Getting-Started-TPU-VM.md

```diff
@@ -43,25 +45,24 @@ Defaults are:
 - `type`: `v3-32`
 - `preemptible`: `false`
 
-The command will spam you with a lot of output, sorry.
-
-In addition to creating the instance, it will set up the venv on each worker, and it will clone the repo to `~/levanter/`
-
-####
-If you use a preemptible instance, you probably want to use the "babysitting" script that automatically re-creates
-the VM. That's explained down below in the "Running Levanter GPT-2" section.
-
+**For Stanford CRFM Users**:
 
-### CRFM Setup
-
-Stanford CRFM folks can pass a different setup script to `infra/spin-up-tpu-vm.sh` to get our NFS automounted:
+Stanford CRFM folks can pass a different setup script to `infra/spin-up-vm.sh` to get our NFS automounted:
 ```bash
-bash infra/spin-up-tpu-vm.sh <name> -z <zone> -t <type> [--preemptible] -s infra/setup-tpu-vm-nfs.sh
+bash infra/spin-up-vm.sh <name> -z <zone> -t <type> [--preemptible] -s infra/setup-tpu-vm-nfs.sh
 ```
 
 In addition to creating the instance, it will also mount the `/files/` nfs share to all workers, which has a good
 venv and a copy of the repo.
 
+**Notes**:
+- This uploads setup scripts via scp. If the ssh-key that you used for Google Cloud requires passphrase or your ssh key
+path is not `~/.ssh/google_compute_engine`, you will need to modify the script.
+- The command will spam you with a lot of output, sorry.
```

Comment:
we should probably add a -q flag or something. I am sorry about the spam

Reply:
The log is helpful for me to know what is going on and what failed. Maybe we can add an argument for showing/hiding the log :)